GeneratingDataset.py

```diff
@@ -1344,18 +1344,20 @@ def create_vocab(cls, **opts):
       clz = BytePairEncoding
     return clz(**opts)
 
-  def __init__(self, vocab_file, unknown_label="UNK", num_labels=None):
+  def __init__(self, vocab_file, unknown_label="UNK", num_labels=None, seq_postfix=None):
```

Comment:
The corresponding option should be removed in all superclasses, e.g. `BytePairEncoding`.